README.md

```diff
@@ -25,15 +25,16 @@ $ cd onlineweb4
 
 To fire up the dev environment, you should use docker-compose.
 
-Install it by running `pip install docker-compose`.
+First install Docker and Docker-compose 
 
-There exists a `Makefile` in the project root directory. This simplifies interaction with docker and docker-compose.
```

Comment:
I think this line should remain. Running `make` commands is the simplest way to interact with docker and OW4 during development, the start and stop commands are pretty simple compared to `docker-compose`-commands.